### PR TITLE
NR-93424 Added api pages for enable and disable feature

### PR DESCRIPTION
**Pull request by mbruin-NR**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

This PR adds api documentation for the enableFeature and disableFeature function for the iOS agent.

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/enable-features.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Set this bit-wise flag to enable features.
+
+## Requirements
+
+Compatible with all agent versions.
+
+## Description
+
+Most of the features in the New Relic iOS agent are enabled by defualt. The `enableFeatures` API can be used if you want to enable those features that are not enabled by defualt.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/enable-features.mdx:26]:
```suggestion
Most of the features in the New Relic iOS agent are enabled by defualt. The `enableFeatures` API can be used if you want to enable those features that are not enabled by default.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/enable-features.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+For more information on which features are enabled by default see [iOS agent configuration and feature flags]((/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags))
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/enable-features.mdx:27]:
```suggestion
For more information on which features are enabled by default see [iOS agent configuration and feature flags](/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags).
```

**Review by ghost — DISMISSED** [review]

**Comment by ally-sassman** [comment]
@mbruin-NR Is this ready to be merged?

**Comment by mbruin-NR** [comment]
Sorry for the delay in response but this can be merge in whenever you are ready now. Thank you!

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/disable-features.mdx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+Set this bit-wise flag to disable features.
+
+## Requirements
+
+Compatible with all agent versions.
+
+## Description
+
+The `disableFeatures` API can be used to disable features that aren't wanted to run.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/disable-features.mdx:26]:
```suggestion
The `disableFeatures` API can be used to disable features you don't want to run. 
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/disable-features.mdx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+For more information on the flags that can be disabled see [iOS agent configuration and feature flags]((/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags))
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/disable-features.mdx:27]:
```suggestion
For more information on the flags that can be disabled see [iOS agent configuration and feature flags](/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags). 
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/disable-features.mdx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+<Callout variant="important">
+This function most be called just before calling:
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/disable-features.mdx:30]:
```suggestion
This function must be called just before calling:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/enable-features.mdx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+Compatible with all agent versions.
+
+## Description
+
+Most of the features in the New Relic iOS agent are enabled by default. The `enableFeatures` API can be used if you want to enable those features that are not enabled by default.
+For more information on which features are enabled by default see [iOS agent configuration and feature flags](/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags).
+
+<Callout variant="important">
+This function most be called just before calling:
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/enable-features.mdx:30]:
```suggestion
This function must be called just before calling:
```

**Review by ghost — APPROVED** [review]

**Pull request merged**